mitodl/open-discussions
# Import content files for mitx platform

#### Pre-Flight checklist


#### What are the relevant tickets?
Closes #2667

#### What's this PR do?
Imports content files from S3 for mitx courses.

#### How should this be manually tested?
- Set `EDX_LEARNING_COURSE_BUCKET_NAME=edxorg-qa-edxapp-courses` and set the other `EDX_*` settings to the same as on RC:
  ```
  EDX_API_URL
  EDX_API_CLIENT_ID
  EDX_API_CLIENT_SECRET
  EDX_API_ACCESS_TOKEN_URL
  ```
- Run `backpopulate_edx_data` followed by `backpopulate_edx_files`.  Both should succeed, but the latter takes close to 2 hours.  You can force-stop it after 10 minutes or so if you don't want to wait that long.
- Check that you have some `LearningResourceRun` objects and `Course` objects with `platform=mitx`, as well as numerous `ContentFile` objects with `run__platform=mitx`
